lib/ansible/modules/cloud/amazon/tag_resources.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, yanan@epicon.com.au
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+import json
+import traceback
+
+try:
+    from botocore.exceptions import BotoCoreError, ClientError
+except ImportError:
+    pass    # Handled by AnsibleAWSModule
+
+from ansible.module_utils.aws.core import AnsibleAWSModule
+
+DOCUMENTATION = '''
+---
+module: tag_resources
+version_added: "2.9"
+short_description: manage AWS resources tags.
+description:
+     - tag or untag AWS resoruce tags.
+     - Check https://docs.aws.amazon.com/ARG/latest/userguide/supported-resources.html for the supported resources
+options:
+  state:
+    description:
+      - Specify the desired state of the tag.
+    choices: [ 'present', 'absent']
+    type: str
+  arns:
+    description:
+      - The resource arns .
+    required: true
+    aliases:
+      - resource_arns
+    type: list
+  tags:
+    description:
+      - tags dict to apply to the resources. Required when the state is present
```

Comment:
```suggestion
      - A dict of tags to apply to the resources.
      - Required when the state is present.
```

Descriptions should be full sentences:
1. Start with a capital letter.
2. End with a full stop.